content/reference/admin/private_locations/kubernetes_deployment/index.md

```diff
@@ -0,0 +1,106 @@
+---
+title: "Deploying a Control Plane on Kubernetes"
+description: "How to deploy a Gatling Control Plane on Kubernetes, to set up your Private Locations and run load generators in your own Kubernetes cluster"
+lead: "Run a Control Plane on Kubernetes, to set up your Private Locations and run load generators in your own Kubernetes network"
+weight: 22053
+---
+
+### Introduction
+
+A control plane, deployed on a kubernetes cluster with the needed roles will be able to deploy kubernetes private locations.
+
+To better understand what roles are needed, here's a quick overview of the mechanisms involved when deploying a private location on Kubernetes.
+
+First, the control plane is notified to deploy a number of instances for a given Kubernetes location.
+
+Then, a config map is created, given the namespace of the kubernetes location, with the starting script of a Gatling load generator.
+*This script is not embedded in our official images, to allow custom images*
+
+Finally, a batch job is initiated, with the number of instances configured on the simulation run, config map is mounted on each pod of the job.
+
+### Roles
+
+For each private location configured namespace, following roles are needed:
+
+```yaml
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: control-plane-role
+  namespace: <namespace>
+rules:
+  - apiGroups: [""]
+    resources: ["configmaps"]
+    verbs: ["get", "create", "update", "delete"]
+  - apiGroups: ["batch"]
+    resources: ["jobs"]
+    verbs: ["create"]
+```
+
+### Example
+
+Control plane require a configuration under `/app/conf/control-plane.conf`, see [control plane configuration]({{< ref "../configuration#control-plane-configuration-1" >}}).
+
+{{< alert tip >}}
+To do so, we'll create a config map named `control-plane-config` with  file under `control-plane.conf`, that will be mounted to the container:
+`kubectl create configmap control-plane-config --from-file=control-plane.conf`
```

Comment:
```suggestion

```bash
kubectl create configmap control-plane-config --from-file=control-plane.conf
```

Reply:
Missing the ending ``` :<